.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Lints and formatting
```

Comment:
Btw, for the error you referenced me when trying to run tests (`%1 is not a valid win32 application" Rust Command run fail`) I discovered that this has to to with `ftd2xx` being a `cdylib`, but specifically with the fact that we emit `cargo:rustc-cdylib-link-arg=/DEF:<...>` in the build script. If I remove that `println!` statement from the builds script, the tests run under windows (there are some import issues under windows but ok).